pkg/services/secrets/manager/manager.go

```diff
@@ -39,15 +41,29 @@ func ProvideSecretsService(
 
 	currentProvider := settings.KeyValue("security", "encryption_provider").MustString(kmsproviders.Default)
 
+	logger := log.New("secrets")
+	if _, ok := providers[currentProvider]; !ok {
+		logger.Error("Configuration for current encryption provider is missing", "provider", currentProvider)
```

Comment:
Minor comment: not sure if both logging and returning an error could imply some repetition. If the function caller's already prints out the error in a log line, it may be enough just returning the error.